src/app/components/pages/login-page/login-page.component.html

```diff
@@ -3,6 +3,10 @@
   <h2 class="login-page__title">
     Se connecter
   </h2>
+  <p class="card card--warning">
```

Comment:
The `--warning` decorator doesn't exist on the `card` SCSS component. It could be great to create it in `src/assets/scss/components/_card.scss` to add some color.